GOVERNANCE.md

```diff
@@ -4,51 +4,170 @@ This document defines governance policies for the Kiali project.
+
+A new maintainer must be proposed by an existing maintainer by opening a pull request
+modifying the maintainers list to add the new proposed maintainer, or via a [GitHub
+discussion](https://github.com/kiali/kiali/discussions/new) under the Governance category.
+The following information must be provided:
+
+* nominee's GitHub username,
+* an explanation of why the nominee should be a maintainer,
+* a list of links to non-trivial pull requests (top 10) authored by the nominee.
 
 Two other maintainers need to second the nomination. If no one objects in 5 working days (U.S.), the nomination is accepted.  If anyone objects or wants more information, the maintainers discuss and usually come to a consensus (within the 5 working days). If issues can't be resolved, there's a simple majority vote among current maintainers.
 
-## Changes in Maintainership
+To upgrade to _Admin_ of a Kiali repository or _Owner_ of the Kiali organization,
+you must first be a Maintainer (i.e. you cannot directly become an _Admin_ or _Owner_
+without being first a Maintainer). You can propose yourself to apply for admin privileges
+or ownership, or another Maintainer can do it for you. Proposals must be posted by opening a pull request
+modifying the admin or owners list to add the new _Admin_ or _Owner_, or via a
+[GitHub discussion](https://github.com/kiali/kiali/discussions/new) under the
+Governance category. The proposal must mention the GitHub username of the maintainer requiring
+the upgrade and the reason of the need of the upgrade (usually, it is because
+of the need of extra privileges). The new privileges are granted if a simple majority vote
+among current maintainers succeeds.
+
+## Testers
+
+Kiali testers dedicate time and resources to ensure that the Kiali project is
+delivered with good quality, by actively trying pull requests to find bugs,
+performance issues and any other kind of issues. Testers may also write manual or
+automated tests. The focus is on "System testing" and "Integration testing",
+although it is possible to do simple sanity, smoke and regression testing, or
+simply running existent automated tests if that is enough for a certain code change.
+
+Testers are granted the same privileges as Maintainers, and are also invited to become
+a member of the Kiali organization. This is in good faith of not performing tasks of a
+Maintainer.
+
+Testers do not need to be Maintainers. Also, Maintainers do not need to be Testers. Yet,
+both roles aren't mutually exclusive.
+
+Current list of testers (alphabetically ordered):
+
+* [mattmahoneyrh](https://github.com/mattmahoneyrh)
+* [pbajjuri20](https://github.com/pbajjuri20)
+* [prachiyadav](https://github.com/prachiyadav)
+* [skondkar](https://github.com/skondkar)
+
+### Becoming a Tester
+
+To become a Tester you need to:
+
+* actively participate in testing code changes of the Kiali project
+  * testing pull requests for 3 months or more,
+  * find 5 non trivial issues in Kiali,
```

Comment:
What does this mean exactly? Finding bugs in the Kiali project?

Reply:
I was thinking issues in the broad sense. It could be performance issues, for example. Typos are also issues, although probably typos fall under the "trivial" space...

Reply:
maybe "defects" as opposed to issues?  Issues also include enhancements, at least in the github sense of 'issue'.

Reply:
@jshaughn OK, I'll use "defets" to avoid confusions.